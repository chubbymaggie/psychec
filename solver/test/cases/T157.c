void f()
{
    union O orphan;
}

void g()
{
    union U u;
    u.w = 1;

    struct ST st;
    st.ww = 1;
}

union P { int xx; };

void h() {
    union P p;
    p.xx = 1;
}

union Q { T x; };

void k() {
    union Q q;
    q.x = 3.14;
}

typedef union S { double y; } S;

void i() {
    S s;
    s.y = 3.14;
}

typedef union J { TT z; } J;

void m() {
    J j;
    j.z = "foo";
}
