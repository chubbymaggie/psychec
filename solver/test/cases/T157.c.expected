/* Forward declarations */

/* Type definitions */
union U {int w; } ;
typedef  union O {int dummy; } O ;
struct ST {int ww; } ;
typedef  char* TT ;
typedef  double T ;

/* Variables and functions */

